src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+
+const ArticlesList = ({ getArticles, setGetArticles }) => {
+  useEffect(() => {
+    axios
+      .get("https://news-v9aq.onrender.com/api/articles")
+      .then((response) => {
```

Comment:
It's good practice to move the code for this request into it's own function in api.js file and then requiring that in rather than just having it out here in the component itself.